charts/spark-operator-chart/values.yaml

```
@@ -105,7 +105,11 @@ controller:
     # key2: value2
 
   # -- Volumes for controller pods.
-  volumes: []
+  volumes:
+    # -- Create a tmp directory to write Spark artifacts to for deployed Spark apps.
```

```suggestion
    # Create a tmp directory to write Spark artifacts to for deployed Spark apps.
```

Remove the `--` so this comment will not rendered in the `README.md` file.

---

I completed these changes locally and pushed them but now realize I could've directly committed these changes from your suggestions.